src/test/kotlin/nl/avisi/structurizr/site/generatr/site/views/CDNTest.kt

```diff
@@ -5,21 +5,26 @@ import assertk.assertThat
 import assertk.assertions.doesNotContain
 import assertk.assertions.endsWith
 import assertk.assertions.startsWith
+import com.structurizr.Workspace
 import org.junit.jupiter.api.DynamicTest
 import org.junit.jupiter.api.TestFactory
 
 class CDNTest {
```

Comment:
Could you please also add tests that 
- Test the default CDN
- Test a CDN location from the view configuration with ending slash
- Test a CDN location from the view configuration without ending slash